gen3users/user_security.py

```diff
@@ -65,9 +55,21 @@ def _check_yaml(
 
     if response.status_code == 401 and (token == "" or token == None):
         raise TokenError("Need to set environment variable USER_YAML_GITHUB_PAT")
-    content = response.content
+    content = response.content.decode()
+    return content
+
+
+def _check_yaml(
+    resource_names,
+    content,
+    verbose=False,
+):
+    raw_content = content
     content = yaml.safe_load(content)
 
+    if type(resource_names) == str:
```

Comment:
I would check if it's _not_ a string and throw an error if not, otherwise proceed with assignment.